static/js/ys/components/QuickCreateClaim.jsx

```
@@ -58,7 +58,7 @@ export default class QuickCreateClaim extends React.Component {
               <TitleText id="newPointTextField" className="titleTextField" placeholder='Make a claim, eg "Dogs are better than cats."' />
             </div>
             <div className="claimCreationFormButtonBlock">
-              {this.submitButton({disabled: (!title || (title == "")) || (validationFailures > 0)})}
+              {this.submitButton({disabled: (!title || (title == "") || (title.length > validations.titleMaxCharacterCount) ) || (validationFailures > 0)})}
```

if `title.length > validations.titleMaxCharacterCount` I think validationFailures should be greater than 0, so this seems redundant... it sounds like you've seen a case where `title.length > validations.titleMaxCharacterCount` but `validationFailures == 0`, which feels like a bug of some sort to me... does that sound right? if so, this isn't the right fix - we should dig into why validationFailures isn't 1 when the title is over 200 chars

---

Yes that was exactly my thought. It looks to me like the validatateTitle function in validations.js is returning its messages when we want it to, so I am guessing this bug is in how validationFailures is declared... but that code is a bit beyond me.

---

It's not a bug. react-form doesn't set validationFailures when a validation returns a simple string. Instead, it sets the `errors` object, so `errors` is what needs to be checked here (in addition to validationFailures, in case our validations ever do return a validation object instead of a string). I'm fixing this in a new PR.

from react-form: "A validation function accepts the value as a parameter and must return either a validation object, where the keys are one of ['error', 'warning', 'success'], and the value is a message or null, OR if you only care about errors you can simply return an error!"